awx/main/management/commands/inventory_import.py

```diff
@@ -893,6 +906,12 @@ def check_license(self):
         if time_remaining <= 0 and not license_info.get('demo', False):
             logger.error(LICENSE_EXPIRED_MESSAGE)
             raise CommandError("License has expired!")
+        # special check for tower-type inventory sources
+        # but only if running the plugin
+        if self.inventory_source.source == 'tower' and ('tower.yml' in self.source or 'tower.yaml' in self.source):
```

Comment:
Don't feel like you have to change this, but I prefer something like this.

    TOWER_SOURCE_FILES = ['tower.yml', 'tower.yaml']
    [f for f in TOWER_SOURCE_FILES if f in self.source]

Reply:
that's relying on `[]` for the false case. maybe `any(f in self.source for f in TOWER_SOURCE_FILES)` would be more readable